### DISCUSSION TITLE
Could not transfer artifact org.jboss.shrinkwrap:shrinkwrap-bom:pom:1.2.6

### DISCUSSION THREAD

**Discussion by karthikrathnam** [oracle/opengrok#4193] [Development]
I'm trying to build 1.8.1 and it always fails while downloading this plugin. Am I missing something here

[ERROR] Failed to execute goal on project opengrok: Could not resolve dependencies for project org.opengrok:opengrok:jar:1.8.1: Failed to collect dependencies at org.glassfish.jersey.ext:jersey-bean-validation:jar:3.1.0 -> org.hibernate.validator:hibernate-validator:jar:8.0.0.Final: Failed to read artifact descriptor for org.hibernate.validator:hibernate-validator:jar:8.0.0.Final: Could not transfer artifact org.jboss.shrinkwrap:shrinkwrap-bom:pom:1.2.6 from/to central (https://repo.maven.apache.org/maven2): Connection reset -> [Help 1]

**Comment by vladak** [comment]
Works for me. I deleted the `shinkwrap` from my local `.m2` cache and did `mvn package` which downloaded the files successfully from  https://repo.maven.apache.org/maven2/org/jboss/shrinkwrap/shrinkwrap-bom/1.2.6/shrinkwrap-bom-1.2.6.pom

The 'connection reset' would hint towards some sort of networking problem or perhaps rate limiting on the server side.

**Reply by karthikrathnam** [reply]
@vladak I use a remote repo in artifactory to get these plugins? will that be a problem? other plugins are being downloaded except this

**Reply by karthikrathnam** [reply]
this command works `mvn dependency:resolve-plugins`

But neither of this work
` mvn -DskipTests -Dcheckstyle.skip -Dmaven.antrun.skip package
`or
` mvn -DskipTests=true -Dmaven.javadoc.skip=true -B -V package
`or
`mvn package
`
I'm not a super expert when it comes to maven, so I'm kind of stuck with this problem :(

**Reply by vladak** [reply]
Me neither. What kind of environment do you run this in ? Some corporate network maybe ?

Not sure what do you mean by "remote repo in artifactory".

**Reply by karthikrathnam** [reply]
I managed to fix this. I was missing settings.xml with mirror repo in it. I will close this topic. But I have another question. I will start that in another thread
